prusti-encoder/src/encoders/mir_pure_function.rs

```diff
@@ -79,8 +79,9 @@ impl TaskEncoder for MirFunctionEnc {
                 .map(|def_idx| local_defs.locals[def_idx].ty.snapshot)
                 .collect();
             let args = UnknownArity::new(vcx.alloc_slice(&args));
-            let function_ref = FunctionIdent::new(function_name, args);
-            deps.emit_output_ref::<Self>(*task_key, MirFunctionEncOutputRef { function_ref, return_type: local_defs.locals[mir::RETURN_PLACE].ty });
+            let return_type = local_defs.locals[mir::RETURN_PLACE].ty;
+            let function_ref = FunctionIdent::new(function_name, args, return_type.snapshot);
+            deps.emit_output_ref::<Self>(*task_key, MirFunctionEncOutputRef { function_ref, return_type });
```

Comment:
Is the `return_type` field of this output ref obsolete now?

Reply:
Yeah it doesn't seem necessary, removed it.